docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,77 @@
+
+These efforts are a part of optimizing our work in supporting and growing the community, and the also aim to keep things fresh and exciting for our documentarians.
+
+We hope that you'll enjoy the shiny new things and of course, if you'd like to contribute to any of these efforts or have experiments that you'd like to pursue, please write to us at support@writethedocs.org and let us know!
+
+So, new year - new...
+
+Core Team
+---------
+
+Since the original Write the Docs conference was held in 2012, the folks behind the scenes have contributed their time, efforts, and talent on a volunteer basis, with tremendous results.
+
+Over the years, the community grew to a point where we needed to examine its long-term needs and develop a system that will support the community without burning out the people who are filling key global roles.
+
+Last year we decided to try setting up a formalized a small, part-time paid staff with the goal of consolidating various roles and responsibilities that could be handled globally, such as web administration, swag coordination, staff wrangling, and meetup overseeing.
+
+So far the experiment has been successful, and the core team now includes the following dedicated documentarians:
+
+- Eric Holscher: Co-founder, head of North America conferences, global sponsorship coordinator
+- Mikey Ariel: Head of Europe conferences, global swag coordinator, communication chair
+- Samuel Wright: Head of support, Webmaster/GitHubmaster, meetup overseer, speaker coordinator (Europe)
+- Kelly O'Brien: Head of newsletter, job board coordinator, speaker coordinator (North America)
+
+These roles are on an including-but-not-limited-to basis, since there are lots of smaller, ad-hoc tasks that the core team runs into throughout the year.
+
+If you want to learn more about the organization behind the community, you can read :doc:`About Us </about-us>`. You can also find a high-level overview of the teams and tasks in :doc:`Team </team>`.
+
+Job Board
+---------
+
+As we mentioned in previous newsletters and Slack posts, we are launching a shiny new job board! Led by Kelly O'Brien, the job board aims to make it easy for documentarians to find their dream gigs, and for employers to find their dream documentarians.
+
+We're currently in the beta-testing phase, so if you're interested in helping us test the board – either as an employer or an applicant – we'd love to hear from you at `jobs@writethedocs.org <mailto:jobs@writethedocs.org>`_.
+
+Note: In order to help optimize our job-related efforts, we looked at our current communication channels, and came to the conclusion that the #jobs Slack channel together with the job board is the most likely to be a successful combination. Therefore, we are sadly closing the Write the Docs forum on (date?).
```

Comment:
Can probably not mention the forum here, since there's an open discussion of the community trying to save it.